bedrock/privacy/views.py

```diff
@@ -80,15 +80,24 @@ def get_legal_doc(self):
         return doc
 
 
+class FirefoxCliqzPrivacyDocView(PrivacyDocView):
+    template_name = 'privacy/notices/firefox-cliqz.html'
+
+    def get_legal_doc(self):
```

Comment:
Should this method have some sort of unit test?

Reply:
This one actually "implements" the function inherited from `PrivacyDocView` so, there should already be a test for that?

Reply:
Hmm, maybe I'm looking in the wrong place - but I can see a test for the original function either?

Reply:
@pmac Thoughts? ^^

Reply:
It would be best if there were a unit test for both of these. I didn't push it because we manually verified it works and we'll be able to delete these work-arounds once all of the docs in the legal-docs repo are updated with the new format. If you'd like to push for tests for these though I can't really fault that logic. It's a function of how quickly we need this to land really.

Reply:
I don't mind merging this without one if that is best, but it feels like we probably should have some form of test? Could make it a followup PR if that works 👍

Reply:
It also feels like we're going to go through this for every Privacy notice, so some form of reusable & testable function may come in handy?

Reply:
Hmm... is that right? If that's true then I'm happy to do a followup PR to clean these up and add a reusable and tested base class for these.

Reply:
Yes I agree. As more of these are converted, there will be more and more that needs this same kinda fallback as some locales are being localized so, having a base class would be great. Thanks @pmac 

I am waiting for the PR on legal-docs to be merged and, for the relevant people to sign of on the page. As soon as I have those, I can remove the DNM label from this one.